float64 screwVel1
float64 screwVel2
float64 jointPos1
float64 jointPos2

